src/playdate/system.nim

```diff
@@ -6,6 +6,7 @@ import sequtils
 
 import bindings/[api, types]
 import bindings/system
+import domain/lcdbitmap {.all.}
```

Comment:
Can you use `privateAccess` instead of `all`?

Reply:
That seems not to work for fields...